### PR TITLE
bugfix: fix bugs after test....

### PR DESCRIPTION
**Pull request by wangzhuzhen**
#### What type of this PR

Add one of the following kinds:
/kind feature
/kind bug
/kind polish
/kind document

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @your-reviewer

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by iutx — APPROVED** [review]
/lgtm

File: erda-helm/erda/charts/sonar/Chart.yaml [hunk 1]
```diff
@@ -5,12 +5,9 @@ type: application
 version: 8.4.2
 keywords:
   - sonar
+#home: https://goharbor.io
+#icon: https://raw.githubusercontent.com/goharbor/website/master/static/img/logos/harbor-icon-color.png
```
Comment (luobily) [inline erda-helm/erda/charts/sonar/Chart.yaml:9]: delete it

File: erda-helm/erda/charts/sonar/templates/sonar-deployment.yaml [hunk 2]
```diff
@@ -22,15 +22,9 @@ spec:
...
-            configMapKeyRef:
-              key: SONAR_ADMIN_PASSWORD
-              name: erda-sonar
+          value: {{ .Values.password }}
         - name: SONAR_ADMIN_TOKEN
-          valueFrom:
-            configMapKeyRef:
-              key: SONAR_ADMIN_TOKEN
-              name: erda-sonar
+          value: {{ .Values.token }}
```
Comment (luobily) [inline erda-helm/erda/charts/sonar/templates/sonar-deployment.yaml:27]: rollback

File: erda-helm/erda/charts/sonar/Chart.yaml [hunk 3]
```diff
@@ -5,12 +5,9 @@ type: application
...
 sources:
   - https://github.com/erda-project/erda-release
 maintainers:
-  - name: Luobingli
-    email: luobily@gmail.com
-  - name: Wangzhuzhen
-    email: zhuzhenwang1984@gmail.com
-  - name: Liutianxiang
-    email: root@viper.run
+  - name: Erda SRE Team
```
Comment (luobily) [inline erda-helm/erda/charts/sonar/Chart.yaml:13]: rollback

File: erda-helm/erda/charts/sonar/Chart.yaml (hunk 1 above)
Comment (luobily) [reply erda-helm/erda/charts/sonar/Chart.yaml:9]: rollback

File: erda-helm/erda/charts/sonar/templates/sonar-deployment.yaml [hunk 4]
```diff
@@ -43,10 +37,13 @@ spec:
...
-        {{- with .Values.resources }}
         resources:
-          {{- toYaml . | nindent 12 }}
-        {{- end }}
+          requests:
+            cpu: {{ .Values.resources.requests.cpu | default "0.1" | quote }}
+            memory: {{ .Values.resources.requests.memory | default "10Mi" | quote }}
+          limits:
+            cpu: {{ .Values.resources.limits.cpu |  default "1.5" | quote }}
+            memory: {{ .Values.resources.limits.memory | default "3Gi" | quote }}
```
Comment (luobily) [inline erda-helm/erda/charts/sonar/templates/sonar-deployment.yaml:46]: rollback

File: erda-helm/erda/charts/registry/values.yaml [hunk 5]
```diff
@@ -1,20 +1,31 @@
...
+  # 镜像参数
   image:
+    # 镜像仓库
     repository: registry.erda.cloud/erda
+    # 镜像拉取策略
     imagePullPolicy: IfNotPresent
+  # 镜像拉取 Secrets
   imagePullSecrets: [ ]
+  # 部署组件中 StatefulSet 负载使用的存储的 StorageClass 对象的名称
+  storageClassName:
```
Comment (luobily) [inline erda-helm/erda/charts/registry/values.yaml:16]: Don't put it in global, storageClass cannot be unified.

**Review by luobily — CHANGES_REQUESTED** [review]

File: erda-helm/erda/charts/sonar/Chart.yaml (hunk 1 above)
Comment (wangzhuzhen) [reply erda-helm/erda/charts/sonar/Chart.yaml:9]: ok

File: erda-helm/erda/charts/registry/values.yaml (hunk 5 above)
Comment (wangzhuzhen) [reply erda-helm/erda/charts/registry/values.yaml:16]: Adjust storageClassName  will in next Stage.

**Comment by wangzhuzhen** [comment]
Resolved after review.

**Review by luobily — APPROVED** [review]

**Pull request merged**